jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -167,6 +170,21 @@ def search(self, error_on_fail=False):
                             'Object ID': row['objid']}
                 self.table.add_item(row_info)
 
+        elif self.catalog_selected == 'Gaia':
+            from astroquery.gaia import Gaia
+
+            Gaia.ROW_LIMIT = self.max_rows
+            sources = Gaia.query_object(skycoord_center, radius=zoom_radius,
+                                        columns=('source_id', 'ra', 'dec'))
+            self.app._catalog_source_table = sources
```

Comment:
Can we make this public in the user API?

Reply:
Is there a reason making `self.app._catalog_source_table` public API belongs in a PR for enabling Gaia catalogs? I'm not really sure why we would do this so genuinely curious.

Reply:
Looking more closely, I realize that this attr is on `app` and not the plugin, and the plugin has a `table` attr – I agree with you, let's not do that. Apologies 🙊